charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -4,7 +4,6 @@ kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: efs-csi-controller
-  namespace: kube-system
```

Comment:
Should this be replaced with {{ .Release.Namespace}}?

Reply:
following the ebs version of the PR, it's fine to leave it blank and it will automagically get created in the namesapce you pass via helm install -n namespace

Reply:
What if they don't pass in the namespace flag and continue with what they've been used to?

Reply:
i tested, it will go to default namespace. or whatever namespace their current context is

Reply:
updated helm upgrade command in README to include namespace flag. same as ebs repo https://github.com/kubernetes-sigs/aws-ebs-csi-driver#deploy-driver